docs/introduction.md

```diff
@@ -7,21 +7,83 @@ analyses. It offers an infrastructure for rapid development of mass
-
-OpenMS offers analyses for various quantitation protocols, including label-free
-quantitation, SILAC, iTRAQ, TMT, SRM, SWATH, etc.
-
-OpenMS provides built-in algorithms for de-novo identification and database search,
-as well as adapters to other state-of-the art tools like X!Tandem, Mascot,
-OMSSA, etc. It supports easy integration of OpenMS built tools into workflow
-engines like KNIME, Galaxy, WS-Pgrade, and TOPPAS via the TOPPtools concept and
-a unified parameter handling via a 'common tool description' (CTD) scheme.
-
-With pyOpenMS, OpenMS offers Python bindings to a large part of the OpenMS API
-to enable rapid algorithm development. OpenMS supports the Proteomics Standard
-Initiative (PSI) formats for MS data. The main contributors of OpenMS are
-currently the Eberhard-Karls-Universität in Tübingen, the Freie Universität
-Berlin, and the ETH Zürich.
+```{note}
+Note: This introduction is aimed at users new to the field of LC-MS data analysis and will introduce some basics terms
+and concepts. How to handle the data analysis, available data structures, algorithms and more are covered in the various
+subsections of this documentation.
+```
+
+# Background
+
+Proteomics and metabolomics are interdisciplinary research fields that study structure, function, and interaction of
+proteins and metabolites. They employ large-scale experimental techniques that allow acquiring data at the level of
+cellular systems to whole organisms. One of the main analytical method to identify, characterize or quantify proteins
+and metabolites is mass spectrometry (MS) combined with chromatographic separation.
+
+In mass spectrometry-based proteomics and metabolomics, biological samples are extracted, prepared, and separated to
+reduce sample complexity. The separated analytes are ionized and measured in the mass spectrometer. Mass and abundance
+of ions are stored in mass spectra and used to identify and quantify the analytes in the sample using computational
+methods. The quantity and identity of analytes can then be used, for instance, in biomarker discovery, medical diagnostics,
+or basic research.
+
+# Liquid Chromatography
+
+LC aims to reduce the complexity of the measured sample by separating analytes based on their physicochemical properties.
+Separating analytes in time ensures that a manageable amount of analytes elute at the same time. In mass
+spectrometry-based proteomics, (high-pressure) liquid chromatographic separation techniques (HPLC) are methods of choice
+to achieve a high degree of separation. In HPLC, peptides are separated on a column. Solved in a pressurized liquid
+(mobile phase) they are pumped through a solid adsorbent material (stationary phase) packet into a capillary column.
+Physicochemical properties of each peptide determine how strongly it interacts with the stationary phase. The most
+commonly HPLC technique in proteomics and metabolomics uses reversed-phase chromatography (RPC) columns. RPC employs a
```

Comment:
The most common

Reply:
The most common HPLC technique in proteomics and metabolomics use reversed-phase chromatography (RPC) columns.